### DISCUSSION TITLE
Any end-to-end encryption, by default, based on open keys?

### DISCUSSION THREAD

**Discussion by br3eze** [balzack/databag#5] [General]
Didn't found any info about this.
Obviously, today it's a must have by default.

I mean, something like "Element" messenger. They have bad code but great ideas.

**Comment by balzack** [comment]
In case you were still following this, i've just released e2e encryption for sealed topics.

**Reply by ninchuka** [reply]
what encryption are you using?

**Reply by balzack** [reply]
The messages are encrypted with a 256bit AES key, which is wrapped and secured with a client side genreated 2k RSA key
